cmd/cosign/cli/verify/verify.go

```diff
@@ -256,14 +282,24 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 				return err
 			}
 		} else {
-			// Verify certificate with chain
-			chain, err := loadCertChainFromFileOrURL(c.CertChain)
-			if err != nil {
-				return err
-			}
-			pubKey, err = cosign.ValidateAndUnpackCertWithChain(cert, chain, co)
-			if err != nil {
-				return err
+			if c.CARoots == "" {
+				// Verify certificate with chain
```

Comment:
Do we also want to read in the chain from `ca-roots` and `ca-intermediates` to verify the cert?

Reply:
[TODO] will check and respond

Reply:
@haydentherapper not sure if this is still relevant - I added a comment to the `co.CARoots != nil` case: `// Verify certificate with root (and if given, intermediate) certificate`.  Both with the chain and the `ca-roots` / `ca-intermediates`, the validation is done with the same function [ValidateAndUnpackCert ](https://pkg.go.dev/github.com/sigstore/cosign/v2/pkg/cosign#ValidateAndUnpackCert), only with the certificate chain there is a step of parsing the file and loading the `CheckOpts`.  Let me know or resolve if this is ok or you would like some changes, please.